src/main/java/org/jabref/gui/copyfiles/CopyFilesResultListDependency.java

```diff
@@ -0,0 +1,22 @@
+package org.jabref.gui.copyfiles;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CopyFilesResultListDependency {
```

Comment:
@tobiasdiez  I have created this wrapper class to inject it

Reply:
If I dont' intilaize the ArrayList with Empty, it will give an NPE in my View Model which leads to the conclusion that this are somehow different objects. I remember having similair problems wit the Sharelatex Integration and ended up making it static. But maybe there is another solution?

Reply:
Oh damn...thats ugly. It is not possible to directly inject the list of results?
You also hit this bug here: https://github.com/AdamBien/afterburner.fx/issues/71 this is why you need the no-arg constructor.

Reply:
I'll have a look at it later.

Reply:
The list can not be directly injected, because List is an interface which of course can not be instantiated...that's where our default injector throws an error 
I tried to add handling for list but then generics are a problem...

Reply:
@tobiasdiez  I found the issue. The "key" of the Hashmap and the name of the with inject annotated variable differed! That's why the data was not passed correct

Reply:
With https://github.com/JabRef/jabref/pull/3334 you should be able to use `@Inject List<whatever> toBeInjected` or other complex types (they do not need to have a no-args constructor anymore). Of course the names of the "key" and the "field" have to coincide to make this work.